src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -62,8 +66,79 @@ public void testShorexProductTypeValidate() {
             softAssert.assertThat(
                     response.getPayload().getProducts().get(i).getProductType().getProductType()
                             .equalsIgnoreCase("SHOREX"))
-                    .as("Product id " + response.getPayload().getProducts().get(i).getProductID() +" is not SHOREX")
+                    .as("Product id " + response.getPayload().getProducts().get(i).getProductID() + " is not SHOREX")
                     .isTrue();
         }
     }
+
+    /**
+     * Checking if offeringDate, day  should be less than or equal to 10 days, from sailing date 20200809
+     *
+     * @throws ParseException
+     */
+    @Test
+    public void testShorexOfferingDateValidate() throws ParseException {
+        response = new RestEngine().getResponseGet(baseURL, headerMap, queryParam)
+                .as(Response.class);
+
+        String actualDateFromLink = "20200809";
```

Comment:
can we take this value from the already defined value in Json -    "sailingIdParamValue": "AL20200809"